franka_gazebo/launch/panda.launch

```diff
@@ -7,6 +7,7 @@
   <arg name="paused"      default="false" doc="Should the simulation directly be stopped at 0s?" />
   <arg name="world"       default="worlds/empty.world" doc="Filename to a SDF World for gazebo to use" />
   <arg name="rviz"        default="false" doc="Should RVIz be launched?" />
+  <arg name="physics"     default="dart"   doc="The physics engine used by gazebo"/> <!--Phyics engines: dart|ode-->
```

Comment:
@gollth, @marcbone I still think having this option is helpful since the dart simulation is more stable for me. We can, of course, also set ODE as the default since some Gazebo services are not yet implemented for the DART simulator.